src/main/java/de/tum/in/www1/exerciseapp/domain/Result.java

```diff
@@ -44,6 +45,20 @@
     @ManyToOne
     private Participation participation;
 
+    // number of results in the participation this result belongs to
```

Comment:
Can you please provide a bit more description about this attribute using the official JavaDoc comment notation? 1-2 sentences what it actually means and that it is used for statistics, etc.